float64 timestamp
string command
string vehicle_ids

